test/sql/storage/catalog/test_view_explicit_aliases.test

```diff
@@ -25,10 +25,9 @@ statement ok
 SELECT * FROM test.v
 
 statement ok
-DROP TABLE test.t
+DROP TABLE test.t CASCADE;
```

Comment:
TODO: this should be tested with cross-catalog views, as those will not have dependencies
So in that case the old behavior still exists where a view still exists even if the source is dropped